Tests/js/views/services/assets/ez-locationviewviewservice-tests.js

```diff
@@ -315,7 +315,8 @@ YUI.add('ez-locationviewviewservice-tests', function (Y) {
         "Should fire `editContentRequest` when receiving an editAction event": function () {
             var languageCode = "fre-FR",
                 contentInfo = new Mock(),
-                contentType = {};
+                contentType = {},
+                that = this;
 
             Mock.expect(contentInfo, {
```

Comment:
I think this `expect` is not needed in this test case and the mocks should be verified at the end of test